### ISSUE TITLE
SaveEvent in a calendar getting deleted after viewing 

### ISSUE THREAD

**Issue by vinay340** [wmcmahan/react-native-calendar-events#265]
I am using this package from last 1 year, i am facing issue after upgrading the react-native project to 0.59.10. When an event is created from application, save Event is returning ID and Event is saved In Phone but when i viewed in Calendar It will get deleted.

### Environment
React Native Environment Info:
    System:
      OS: macOS 10.14.5
      CPU: (8) x64 Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
      Memory: 1006.73 MB / 16.00 GB
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 10.15.1 - /usr/local/bin/node
      npm: 6.4.1 - /usr/local/bin/npm
    SDKs:
      iOS SDK:
        Platforms: iOS 12.4, macOS 10.14, tvOS 12.4, watchOS 5.3
      Android SDK:
        API Levels: 23, 25, 26, 27, 28
        Build Tools: 28.0.2, 28.0.3
        System Images: android-23 | Google APIs Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-28 | Google Play Intel x86 Atom
    IDEs:
      Android Studio: 3.4 AI-183.6156.11.34.5692245
      Xcode: 10.3/10G8 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.8.3 => 16.8.3 
      react-native: 0.59.10 => 0.59.10 
    npmGlobalPackages:
      react-native-cli: 2.0.1

### Steps to Reproduce
Using the latest code from Master branch, I tried to add one event using below code

```
RNCalendarEvents.authorizationStatus().then((status)=>{
      console.log('status',status)
      if(status == 'authorized'){
        RNCalendarEvents.saveEvent('Today evevnt', {
          startDate:new Date('2019-08-07T16:00:00.0000z'),
          endDate: new Date('2019-08-07T17:00:00.0000z')
        }).then((data)=>{
          console.log("data",data)
        })
      }
    })
```
Its added successfully in Calendar, But when i open the calendar its disappear in few seconds

### Expected Behavior
Event should be there always until we delete

### Actual Behavior
Its disappear immediately when we open the calendar.

**Comment by vinay340** [comment]
> Issue got resolved by changing the date format to ISO string.

Here is the solution 

`try {
            RNCalendarEvents.saveEvent('Today evevnt 15', {
              calendarId: "calendar id",
              startDate: '2019-08-14T14:48:00.000Z',
              endDate: '2019-08-14T15:48:00.000Z'
            }).then((data) => {
             console.log("Event saved")
            }).catch((err) => {
              console.log(err)
            })
          }
          catch (e) {
           console.log(e)
          }`

**Comment by dexbytes** [comment]
we have also faced same issue . Can you please help as soon as possible. Thank you

**Comment by jitendra8288** [comment]
we have also faced same issue . Can you please help as soon as possible. Thank you

**Comment by vinay340** [comment]
Hi @jitendra8288 , Issue got resolved by changing the date format(i.e, toIsoString()).

It helps your issue.

Thanks

**Comment by jitendra8288** [comment]
Already used. but still not working

**Comment by hardiknariya** [comment]
{ sync: true }, is still not working for me. I cannot see my event in to google calendar. Also, I have tried with calendar sync on/off.  

I have checked my event config, calendarId is primary and allowed modify.  Can anyone help me. 

When I fetch all events for configured time then I can see my event, but not able to see event into google calendar app. For me google calendar is default configured app for calendar app in my phone.
